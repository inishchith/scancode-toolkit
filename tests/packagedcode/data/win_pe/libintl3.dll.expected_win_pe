{
  "APIVersion": "", 
  "Assembly Version": "", 
  "BuildDate": "", 
  "BuildVersion": "", 
  "Comments": "", 
  "Company": "", 
  "CompanyName": "GNU <www.gnu.org>", 
  "FileDescription": "GetText: library and tools for native language support", 
  "FileVersion": "0.14.4.1952", 
  "Full Version": "", 
  "InternalName": "libintl3", 
  "LegalCopyright": "2005 Free Software Foundation <www.fsf.org>", 
  "LegalTrademarks": "GNU, GetText, libintl3", 
  "LegalTrademarks1": "", 
  "LegalTrademarks2": "", 
  "License": "This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License;see www.gnu.org/copyleft/gpl.html.", 
  "OriginalFilename": "libintl3.dll", 
  "ProductName": "GetText", 
  "ProductVersion": "0.14.4.1952", 
  "WWW": "http://www.gnu.org/software/gettext/gettext.html", 
  "extra_data": {
    "LibToolFileVersion": "7:0:4", 
    "SpecialBuild": "GNU for Win32 <gnuwin32.sourceforge.net>"
  }
}