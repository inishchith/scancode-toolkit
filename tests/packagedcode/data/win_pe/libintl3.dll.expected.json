{
  "APIVersion": null, 
  "Assembly Version": null, 
  "BuildDate": null, 
  "BuildVersion": null, 
  "Comments": null, 
  "Company": null, 
  "CompanyName": "GNU <www.gnu.org>", 
  "FileDescription": "GetText: library and tools for native language support", 
  "FileVersion": "0.14.4.1952", 
  "Full Version": null, 
  "InternalName": "libintl3", 
  "LegalCopyright": "2005 Free Software Foundation <www.fsf.org>", 
  "LegalTrademarks": "GNU, GetText, libintl3", 
  "LegalTrademarks1": null, 
  "LegalTrademarks2": null, 
  "License": "This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License;see www.gnu.org/copyleft/gpl.html.", 
  "OriginalFilename": "libintl3.dll", 
  "ProductName": "GetText", 
  "ProductVersion": "0.14.4.1952", 
  "WWW": "http://www.gnu.org/software/gettext/gettext.html", 
  "extra_data": {
    "LibToolFileVersion": "7:0:4", 
    "SpecialBuild": "GNU for Win32 <gnuwin32.sourceforge.net>"
  }
}